.github/workflows/build-and-publish-images.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish Dev Operator
+
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+      - 'v*.*.*-*'
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and publish k8s-operator image
+        env:
+          REPO: ghcr.io/${{ github.repository_owner }}/tailscale/k8s-operator
```

Comment:
Hmm does that really work? GitHub registry images are in the format of `ghcr.io/NAMESPACE/IMAGE_NAME` so the image name would need to allow a `/` in it?

Reply:
ah, well then it might not work :-) Will change it.

Reply:
changed it. Will something like `tailscale-k8s-operator` work as the image name?

Reply:
I guess we'll just have to test it to find out :smile: